subsys/bluetooth/controller/CMakeLists.txt

```diff
@@ -173,10 +173,6 @@ zephyr_library_sources_ifdef(
 zephyr_library_include_directories(
   .
   include
-  )
-
-zephyr_library_include_directories_ifdef(
-  CONFIG_BT_CTLR_CRYPTO
```

Comment:
The commit for this change just says "fix" without any additional information. 

Why are now always including crypto, even if it is disabled?

Reply:
as `IS_ENABLED` is used in the other places that have calls to Host crypto interface, it makes it necessary to always include the prototypes in such files.